src/inputs/Table/components/TableToolbar/style.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import makeStyles from '@mui/styles/makeStyles';
+
+export const FILTER_ROW_HEIGHT = '32px';
+
+export const useStyles = makeStyles((theme) => ({
+  filterRow: {
+    border: 'var(--ag-borders) var(--ag-border-color)',
+    borderBottom: 'none',
+    padding: '8px 16px',
```

Comment:
it's better to use theme padding instead of hardcoding px :slightly_smiling_face:

Reply:
Done